Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -83,10 +91,12 @@ import java.time.Duration
 @Composable
 fun PlayerScreen(
     viewModel: PlayerViewModel,
+    foldableInfo: StateFlow<FoldableInfo>,
```

Comment:
"To expose streams of data, prefer State over StateFlow and LiveData". Is that relevant here?

Reply:
My original implementation used a `State<FoldableInfo`. But after talking with eng about this, they prefer to have it exposed as a `StateFlow` because it comes from the `Activity` and this implementation can be used for View apps and in the interop scenario.